apps/pay/prj.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_LOG=y
 CONFIG_LOG_DEFAULT_LEVEL=3
+CONFIG_COMPILER_WARNINGS_AS_ERRORS=y
```

Comment:
A new line at EOF is missing. 
POSIX defines text files as ending with a newline, so it's good practice to include it.